www/searchutil.php

```diff
@@ -204,7 +204,7 @@ function doSearch($db, $term, $searchType, $sortby, $limit, $browse)
     {
         // special keywords for tag search
         $specialMap = array(
-            "tag:" => array("tag", 99));        
+            "tag:" => array("tag", 0));        

```

Comment:
What's this about?

Reply:
Yeah, we should probably assign constant names to those magic numbers to make the code more readable.

I was using 99 for special-case handling, playing around with different ideas for the tag search. Missed this when I was cleaning up after myself. The 0 is for text matches.